crowbartools/Firebot
# Me chat effect

### Description of the Change

Added a new option to the Chat effect to apply `/me` to the Chat message or Whisper.

### Applicable Issues

[#2145](https://github.com/crowbartools/Firebot/issues/2145)

### Testing

Validated in both Chat messages and Whispers.

1. Add the Chat effect to a trigger (Command, Event, Effect List, etc)
2. Add whatever text you would like "This is a chat message" with or without a variable "This is a message today $date"
3. Check the "Use '/me'" checkbox
4. Trigger the effect and review your chat or whisper

### Screenshots

<img width="586" alt="Screenshot 2024-03-03 at 10 52 50 AM" src="https://github.com/crowbartools/Firebot/assets/10332351/da05657a-6552-4a30-9299-fb51040588a7">
